src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -36,7 +36,9 @@ const initialState = {
   },
 };
 
-const AddInstructors = ({ isOpen, onClose }) => {
+const AddInstructors = ({
```

Comment:
This component is handling different scenarios more than just add a new instructor, so I would suggest to update the name for the file and the component, for something more meaning full, like `InstructorModal` or `InstructorForm`.